LRQA-77402 Add test to cover rollback of files in conjunction with the fileEntries within a transaction

kyle-miho opened the pull request.

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

kyle-miho commented:
@rafaprax I fixed and added test cases for rolling back failed transactions for update and deletion as well, since Dave also mentioned those possibilities. I'm most unsure with the file content update rollback, regarding optimal logic and SF if you can look at that one more. (It seems like the test correctly tests the functionality, but maybe not in a clean or optimal way)

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-31.liferay.com/job/test-portal-source-format/3931/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-31.liferay.com/userContent/jobs/test-portal-source-format/builds/3931/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d5dbe3e5e51aa86f8c5cba36ef367fb570441425">d5dbe3e5e51aa86f8c5cba36ef367fb570441425</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/kyle-miho/liferay-portal/tree/master-qa-77402">master-qa-77402</a><br/>Branch GIT ID: <a href="https://github.com/kyle-miho/liferay-portal/commit/ca3ccf35fa3eeb884f597038a3e9c64ca2c374e0">ca3ccf35fa3eeb884f597038a3e9c64ca2c374e0</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-31.liferay.com/job/test-portal-source-format/3931/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-31.liferay.com/userContent/jobs/test-portal-source-format/builds/3931/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-31.liferay.com/job/test-portal-source-format/3931/">test-portal-source-format#3931</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-31.liferay.com/userContent/jobs/test-portal-source-format/builds/3931/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-core-infra/liferay-portal/pull/2208">liferay-core-infra#2208</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2016077220">[master] ci:test:sf - kyle-miho &gt; liferay-core-infra - PR#2208 - 2022-10-25[14:20:08]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-31.liferay.com/job/publish-testray-report/14497/">publish-testray-report#14497</a><br/></div></div>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/5193/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5193/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:stable - 26 out of 28 jobs passed</h3><h3>:x: ci:test:relevant - 110 out of 117 jobs passed in 4 hours 33 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d5dbe3e5e51aa86f8c5cba36ef367fb570441425">d5dbe3e5e51aa86f8c5cba36ef367fb570441425</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d5dbe3e5e51aa86f8c5cba36ef367fb570441425">d5dbe3e5e51aa86f8c5cba36ef367fb570441425</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2015982868">EE Development Acceptance (master) - 3945 - 2022-10-25[10:27:21]</a></p><details><summary><strong>ci:test:stable - 26 out of 28 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li>central-requirements-jdk8<ul><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/115535/">central-requirements-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/293310/">functional-smoke-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/114891/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/308518/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/1</a></li></ul></li><li>js-unit-jdk8<ul><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/111757/"><strike>js-unit-jdk8/0/0</strike></a></li></ul></li><li>modules-compile-jdk8<ul><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/116692/">modules-compile-jdk8/0/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/106096/">modules-compile-jdk8/0/1</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/253481/">modules-compile-jdk8/0/2</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/116579/">modules-compile-jdk8/0/3</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/113374/">modules-compile-jdk8/0/4</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/125018/">modules-compile-jdk8/0/5</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/117828/">modules-compile-jdk8/0/6</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/100786/">modules-compile-jdk8/0/7</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/116581/">modules-compile-jdk8/0/8</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/100785/">modules-compile-jdk8/0/9</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/290133/">modules-compile-jdk8/0/10</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/290134/">modules-compile-jdk8/0/11</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277583/">modules-compile-jdk8/0/12</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/282324/">modules-compile-jdk8/0/13</a></li></ul></li><li>modules-integration-mysql57-jdk8_stable<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/293307/">modules-integration-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277586/">modules-integration-mysql57-jdk8_stable/0/1</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/275553/">modules-integration-mysql57-jdk8_stable/0/2</a></li></ul></li><li>modules-unit-jdk8_stable<ul><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/117827/">modules-unit-jdk8_stable/0/0</a></li></ul></li><li>pmd-jdk8<ul><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/104313/">pmd-jdk8/0/0</a></li></ul></li><li>portal-license-smoke-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/298080/"><strike>portal-license-smoke-jdk8/0/0</strike></a></li></ul></li><li>rest-builder-jdk8<ul><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/92422/">rest-builder-jdk8/0/0</a></li></ul></li><li>service-builder-jdk8<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/248875/">service-builder-jdk8/0/0</a></li></ul></li><li>unit-jdk8_stable<ul><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/96592/">unit-jdk8_stable/0/0</a></li></ul></li></ul></div><details><summary><strong>26 Successful Jobs:</strong></summary><ul/></details></details><details><summary><strong>ci:test:relevant - 110 out of 117 jobs PASSED</strong></summary><div><h4>7 Failed Jobs:</h4><ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/5193/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>central-requirements-jdk8<ul><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/115535/">central-requirements-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk11_open<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/290135/">functional-smoke-tomcat90-mysql57-jdk11_open/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/293310/">functional-smoke-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb104-jdk8<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/282328/">functional-smoke-wildfly180-mariadb104-jdk8/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb106-jdk8<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/282323/">functional-smoke-wildfly180-mariadb106-jdk8/0/0</a></li></ul></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/298085/">functional-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/265174/">functional-tomcat90-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/275552/">functional-tomcat90-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/230746/">functional-tomcat90-mysql57-jdk8/0/3</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/290137/">functional-tomcat90-mysql57-jdk8/0/4</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/149712/">functional-tomcat90-mysql57-jdk8/0/5</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/282327/">functional-tomcat90-mysql57-jdk8/0/6</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/308520/">functional-tomcat90-mysql57-jdk8/0/7</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/76367/">functional-tomcat90-mysql57-jdk8/0/8</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/298082/">functional-tomcat90-mysql57-jdk8/1/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/230743/">functional-tomcat90-mysql57-jdk8/1/1</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/282325/">functional-tomcat90-mysql57-jdk8/1/2</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/298083/">functional-tomcat90-mysql57-jdk8/1/3</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/104312/">functional-tomcat90-mysql57-jdk8/1/4</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/253478/">functional-tomcat90-mysql57-jdk8/1/5</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277584/">functional-tomcat90-mysql57-jdk8/1/6</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/105025/">functional-tomcat90-mysql57-jdk8/1/7</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/114994/">functional-tomcat90-mysql57-jdk8/1/8</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/230744/">functional-tomcat90-mysql57-jdk8/1/9</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/103013/">functional-tomcat90-mysql57-jdk8/1/10</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/105028/">functional-tomcat90-mysql57-jdk8/1/11</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/115534/">functional-tomcat90-mysql57-jdk8/1/12</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/113375/">functional-tomcat90-mysql57-jdk8/1/13</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/248876/">functional-tomcat90-mysql57-jdk8/1/14</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/76368/">functional-tomcat90-mysql57-jdk8/1/15</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/115756/">functional-tomcat90-mysql57-jdk8/1/16</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/92421/">functional-tomcat90-mysql57-jdk8/1/17</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/100784/">functional-tomcat90-mysql57-jdk8/1/18</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/114892/">functional-tomcat90-mysql57-jdk8/1/19</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/109627/">functional-tomcat90-mysql57-jdk8/1/20</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/100186/">functional-tomcat90-mysql57-jdk8/1/21</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/105026/">functional-tomcat90-mysql57-jdk8/1/22</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/115533/">functional-tomcat90-mysql57-jdk8/1/23</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/118632/">functional-tomcat90-mysql57-jdk8/1/24</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/116584/">functional-tomcat90-mysql57-jdk8/2/0</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/114893/">functional-tomcat90-mysql57-jdk8/2/1</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/104314/">functional-tomcat90-mysql57-jdk8/2/2</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/308521/">functional-tomcat90-mysql57-jdk8/2/3</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/117829/">functional-tomcat90-mysql57-jdk8/2/4</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/100787/">functional-tomcat90-mysql57-jdk8/2/5</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/265177/">functional-tomcat90-mysql57-jdk8/2/6</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/109628/">functional-tomcat90-mysql57-jdk8/2/7</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/149713/">functional-tomcat90-mysql57-jdk8/2/8</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/298081/">functional-tomcat90-mysql57-jdk8/2/9</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/116691/">functional-tomcat90-mysql57-jdk8/2/10</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/118631/">functional-tomcat90-mysql57-jdk8/2/11</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/298084/">functional-tomcat90-mysql57-jdk8/2/12</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/230745/">functional-tomcat90-mysql57-jdk8/2/13</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/293306/">functional-tomcat90-mysql57-jdk8/2/14</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/308519/">functional-tomcat90-mysql57-jdk8/2/15</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/253480/"><strike>functional-tomcat90-mysql57-jdk8/2/16</strike></a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/114995/">functional-tomcat90-mysql57-jdk8/2/17</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277582/">functional-tomcat90-mysql57-jdk8/2/18</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/253477/">functional-tomcat90-mysql57-jdk8/2/19</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/100185/">functional-tomcat90-mysql57-jdk8/2/20</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/109626/">functional-tomcat90-mysql57-jdk8/2/21</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/293305/">functional-tomcat90-mysql57-jdk8/2/22</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/253479/">functional-tomcat90-mysql57-jdk8/2/23</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/275551/">functional-tomcat90-mysql57-jdk8/2/24</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/105027/">functional-tomcat90-mysql57-jdk8/3/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/103012/">functional-tomcat90-mysql57-jdk8/3/1</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/116582/">functional-tomcat90-mysql57-jdk8/3/2</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/116580/">functional-tomcat90-mysql57-jdk8/3/3</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/293309/">functional-tomcat90-mysql57-jdk8/3/4</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/116583/">functional-tomcat90-mysql57-jdk8/3/5</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/96591/">functional-tomcat90-mysql57-jdk8/3/6</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/293308/">functional-tomcat90-mysql57-jdk8/3/7</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/282326/">functional-upgrade-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/290136/">functional-upgrade-tomcat90-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/106095/">functional-upgrade-tomcat90-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/115532/">functional-upgrade-tomcat90-mysql57-jdk8/0/3</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/114891/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/308518/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/1</a></li></ul></li><li>js-unit-jdk8<ul><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/111757/"><strike>js-unit-jdk8/0/0</strike></a></li></ul></li><li>modules-compile-jdk8<ul><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/116692/">modules-compile-jdk8/0/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/106096/">modules-compile-jdk8/0/1</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/253481/">modules-compile-jdk8/0/2</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/116579/">modules-compile-jdk8/0/3</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/113374/">modules-compile-jdk8/0/4</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/125018/">modules-compile-jdk8/0/5</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/117828/">modules-compile-jdk8/0/6</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/100786/">modules-compile-jdk8/0/7</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/116581/">modules-compile-jdk8/0/8</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/100785/">modules-compile-jdk8/0/9</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/290133/">modules-compile-jdk8/0/10</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/290134/">modules-compile-jdk8/0/11</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277583/">modules-compile-jdk8/0/12</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/282324/">modules-compile-jdk8/0/13</a></li></ul></li><li>modules-integration-mysql57-jdk8<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251975/">modules-integration-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/76370/">modules-integration-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/308522/">modules-integration-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/275556/">modules-integration-mysql57-jdk8/0/3</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/275555/">modules-integration-mysql57-jdk8/0/4</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/265176/"><strike>modules-integration-mysql57-jdk8/0/5</strike></a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251973/"><strike>modules-integration-mysql57-jdk8/0/6</strike></a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/275554/"><strike>modules-integration-mysql57-jdk8/0/7</strike></a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/76369/">modules-integration-mysql57-jdk8/0/8</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251974/">modules-integration-mysql57-jdk8/0/9</a></li></ul></li><li>modules-integration-mysql57-jdk8_stable<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/293307/">modules-integration-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277586/">modules-integration-mysql57-jdk8_stable/0/1</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/275553/">modules-integration-mysql57-jdk8_stable/0/2</a></li></ul></li><li>modules-unit-jdk8<ul><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/125019/">modules-unit-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8_stable<ul><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/117827/">modules-unit-jdk8_stable/0/0</a></li></ul></li><li>modules-unit-project-templates-jdk8<ul><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/111758/">modules-unit-project-templates-jdk8/0/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/265175/">modules-unit-project-templates-jdk8/0/1</a></li></ul></li><li>pmd-jdk8<ul><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/104313/">pmd-jdk8/0/0</a></li></ul></li><li>portal-license-smoke-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/298080/"><strike>portal-license-smoke-jdk8/0/0</strike></a></li></ul></li><li>rest-builder-jdk8<ul><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/92422/">rest-builder-jdk8/0/0</a></li></ul></li><li>service-builder-jdk8<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/248875/">service-builder-jdk8/0/0</a></li></ul></li><li>unit-jdk8<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277585/">unit-jdk8/0/0</a></li></ul></li><li>unit-jdk8_stable<ul><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/96592/">unit-jdk8_stable/0/0</a></li></ul></li></ul></div><details><summary><strong>110 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5193/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251973//consoleText">modules-integration-mysql57-jdk8/0/6</a><pre>     [exec] * What went wrong:
     [exec] Gradle build daemon disappeared unexpectedly (it may have been killed or may have crashed)
     [exec] 
     [exec] * Try:
     [exec] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.launcher.daemon.client.DaemonDisappearedException: Gradle build daemon disappeared unexpectedly (it may have been killed or may have crashed)
     [exec]   at org.gradle.launcher.daemon.client.DaemonClient.handleDaemonDisappearance(DaemonClient.java:249)
     [exec]   at org.gradle.launcher.daemon.client.DaemonClient.monitorBuild(DaemonClient.java:222)
     [exec]   at org.gradle.launcher.daemon.client.DaemonClient.executeBuild(DaemonClient.java:184)
     [exec]   at org.gradle.launcher.daemon.client.DaemonClient.execute(DaemonClient.java:147)
     [exec]   at org.gradle.launcher.daemon.client.DaemonClient.execute(DaemonClient.java:98)
     [exec]   at org.gradle.launcher.cli.RunBuildAction.run(RunBuildAction.java:57)
     [exec]   at org.gradle.internal.Actions$RunnableActionAdapter.execute(Actions.java:212)
     [exec]   at org.gradle.launcher.cli.DefaultCommandLineActionFactory$ParseAndBuildAction.execute(DefaultCommandLineActionFactory.java:275)
     [exec]   at org.gradle.launcher.cli.DefaultCommandLineActionFactory$ParseAndBuildAction.execute(DefaultCommandLineActionFactory.java:247)
     [exec]   at org.gradle.launcher.cli.DebugLoggerWarningAction.execute(DebugLoggerWarningAction.java:82)
     [exec]   at org.gradle.launcher.cli.DebugLoggerWarningAction.execute(DebugLoggerWarningAction.java:30)
     [exec]   at org.gradle.launcher.cli.WelcomeMessageAction.execute(WelcomeMessageAction.java:92)
     [exec]   at org.gradle.launcher.cli.WelcomeMessageAction.execute(WelcomeMessageAction.java:38)
     [exec]   at org.gradle.launcher.cli.NativeServicesInitializingAction.execute(NativeServicesInitializingAction.java:44)
     [exec]   at org.gradle.launcher.cli.NativeServicesInitializingAction.execute(NativeServicesInitializingAction.java:26)
     [exec]   at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionReportingAction.java:41)
     [exec]   at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionReportingAction.java:26)
     [exec]   at org.gradle.launcher.cli.DefaultCommandLineActionFactory$WithLogging.execute(DefaultCommandLineActionFactory.java:240)</pre></div></li><li><div><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/298080//consoleText">portal-license-smoke-jdk8/0/0</a><pre>[beanshell] Average thread duration: 222ms
[beanshell] Busy slaves:0
[beanshell] Finished slaves:1
[beanshell] Target slaves:1
[beanshell] Total duration: 222
[beanshell] Remote execution completed in 222ms.
[beanshell] Remote execution starting with 1 threads.
[beanshell] Executing commands: ssh -o NumberOfPasswordPrompts=0 root@test-1-38 'mkdir -p "/mnt/mfs-hdd1-10.0.10/jenkins/testray-results/production/logs/2022-10/test-1-30/test-portal-acceptance-pullrequest(master)/5193/portal-license-smoke-jdk8/0/0"'
[beanshell] Average thread duration: 222ms
[beanshell] Busy slaves:0
[beanshell] Finished slaves:1
[beanshell] Target slaves:1
[beanshell] Total duration: 223
[beanshell] Remote execution completed in 223ms.
[beanshell] Executing commands: rsync -aqz --chmod=go=rx "/opt/dev/projects/github/liferay-jenkins-ee/testray/prepared_rsync_logs"/* "test-1-38::testray-results/production/logs/"
[beanshell] Uploaded https://testray.liferay.com/reports/production/logs/2022-10/test-1-30/test-portal-acceptance-pullrequest(master)/5193/portal-license-smoke-jdk8/0/0/jenkins-console.txt.gz
[beanshell] INFO: Using GOOGLE_APPLICATION_CREDENTIALS=/root/.gcloud/ci-testray.json
[beanshell] Unable to delete because file does not exist /opt/dev/projects/github/liferay-jenkins-ee/testray/prepared_s3_logs
[beanshell] Created S3 Object https://storage.cloud.google.com/testray-results/2022-10/test-1-30/test-portal-acceptance-pullrequest(master)/5193/portal-license-smoke-jdk8/0/0/jenkins-console.txt.gz in 272 ms
[stopwatch] [run.current.job: 3:48.790 sec]
[beanshell] Created cache file in /tmp/jenkins-cached-files/1014646203.txt

BUILD FAILED
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:15012: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-portal/build-test-batch.xml:6658: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-portal/build-test-batch.xml:6692: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-portal/build-test.xml:12072: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-release-tool-ee/build-release-slim.xml:25: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-release-tool-ee/build-release-slim.xml:192: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-portal/build.xml:956: exec returned: 128</pre></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at d5dbe3e5e51aa86f8c5cba36ef367fb570441425:</strong></summary><ol><li><div><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/111757//consoleText">js-unit-jdk8/0/0</a><pre>     [exec] * What went wrong:
     [exec] Execution failed for task ':apps:layout:layout-content-page-editor-web:packageRunTest'.
     [exec] &gt; Process 'command '/opt/dev/projects/github/liferay-portal/build/node/bin/node'' finished with non-zero exit value 1
     [exec] 
     [exec] * Try:
     [exec] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':apps:layout:layout-content-page-editor-web:packageRunTest'.
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda$executeIfValid$3(ExecuteActionsTaskExecuter.java:186)
     [exec]   at org.gradle.internal.Try$Failure.ifSuccessfulOrElse(Try.java:268)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:184)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:173)
     [exec]   at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:109)
     [exec]   at org.gradle.api.internal.tasks.execution.FinalizePropertiesTaskExecuter.execute(FinalizePropertiesTaskExecuter.java:46)
     [exec]   at org.gradle.api.internal.tasks.execution.ResolveTaskExecutionModeExecuter.execute(ResolveTaskExecutionModeExecuter.java:62)
     [exec]   at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:57)
     [exec]   at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:56)
     [exec]   at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:36)
     [exec]   at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.executeTask(EventFiringTaskExecuter.java:77)
     [exec]   at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:55)
     [exec]   at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:52)
     [exec]   at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:200)</pre></div></li><li><div><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/253480//consoleText">functional-tomcat90-mysql57-jdk8/2/16</a><ol><li><div><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/253480//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_DMPermissions_ViewFolderAsOrganizationUser_">LocalFile.DMPermissions#ViewFolderAsOrganizationUser</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-10/test-1-30/test-portal-acceptance-pullrequest(master)/5193/functional-tomcat90-mysql57-jdk8/2/16/LocalFile.DMPermissions_ViewFolderAsOrganizationUser/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-10/test-1-30/test-portal-acceptance-pullrequest(master)/5193/functional-tomcat90-mysql57-jdk8/2/16/LocalFile.DMPermissions_ViewFolderAsOrganizationUser/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-10/test-1-30/test-portal-acceptance-pullrequest(master)/5193/functional-tomcat90-mysql57-jdk8/2/16/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li><li><div><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/265176//consoleText">modules-integration-mysql57-jdk8/0/5</a><ol><li><div><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/265176//testReport/com.liferay.document.library.service.test/DLFileEntryTypeFinderTest/testFilterCountByKeywordsAsPowerUser">com.liferay.document.library.service.test.DLFileEntryTypeFinderTest.testFilterCountByKeywordsAsPowerUser</a><pre>java.lang.AssertionError: expected:&lt;1&gt; but was:&lt;0&gt;
  at org.junit.Assert.fail(Assert.java:89)
  at org.junit.Assert.failNotEquals(Assert.java:835)
  at org.junit.Assert.assertEquals(Assert.java:647)
  at org.junit.Assert.assertEquals(Assert.java:633)
  at com.liferay.document.library.service.test.DLFileEntryTypeFinderTest.testFilterCountByKeywordsAsPowerUser(DLFileEntryTypeFinderTest.java:124)
  at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  at java.lang.reflect.Method.invoke(Method.java:498)
  at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$3.evaluate(TestExecutorRunnable.java:353)
  at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
  at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
  at com.liferay.portal.test.rule.TransactionalTestRule$1.lambda$evaluate$0(TransactionalTestRule.java:125)
  at com.liferay.portal.spring.transaction.BaseTransactionExecutor.execute(BaseTransactionExecutor.java:37)
  at com.liferay.portal.spring.transaction.TransactionInvokerImpl.invoke(TransactionInvokerImpl.java:39)
  at com.liferay.portal.kernel.transaction.TransactionInvokerUtil.invoke(TransactionInvokerUtil.java:28)
  at com.liferay.portal.test.rule.TransactionalTestRule$1.evaluate(TransactionalTestRule.java:120)
...</pre></div></li><li><div><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/265176//testReport/com.liferay.document.library.service.test/DLFileEntryTypeFinderTest/testFilterFindByKeywordsAsPowerUser">com.liferay.document.library.service.test.DLFileEntryTypeFinderTest.testFilterFindByKeywordsAsPowerUser</a><pre>java.lang.AssertionError: [] expected:&lt;1&gt; but was:&lt;0&gt;
  at org.junit.Assert.fail(Assert.java:89)
  at org.junit.Assert.failNotEquals(Assert.java:835)
  at org.junit.Assert.assertEquals(Assert.java:647)
  at com.liferay.document.library.service.test.DLFileEntryTypeFinderTest.testFilterFindByKeywordsAsPowerUser(DLFileEntryTypeFinderTest.java:256)
  at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  at java.lang.reflect.Method.invoke(Method.java:498)
  at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$3.evaluate(TestExecutorRunnable.java:353)
  at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
  at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
  at com.liferay.portal.test.rule.TransactionalTestRule$1.lambda$evaluate$0(TransactionalTestRule.java:125)
  at com.liferay.portal.spring.transaction.BaseTransactionExecutor.execute(BaseTransactionExecutor.java:37)
  at com.liferay.portal.spring.transaction.TransactionInvokerImpl.invoke(TransactionInvokerImpl.java:39)
  at com.liferay.portal.kernel.transaction.TransactionInvokerUtil.invoke(TransactionInvokerUtil.java:28)
  at com.liferay.portal.test.rule.TransactionalTestRule$1.evaluate(TransactionalTestRule.java:120)
  at com.liferay.portal.kernel.test.rule.AbstractT...</pre></div></li></ol></div></li><li>...</li></ol></details></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-30/jobs/test-portal-acceptance-pullrequest(master)/builds/5193/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-30/jobs/test-portal-acceptance-pullrequest(master)/builds/5193/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-30/jobs/test-portal-acceptance-pullrequest(master)/builds/5193/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/5193/">test-portal-acceptance-pullrequest(master)#5193</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5193/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-core-infra/liferay-portal/pull/2208">liferay-core-infra#2208</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2016812195">[master] ci:test:relevant - kyle-miho &gt; liferay-core-infra - PR#2208 - 2022-10-25[15:40:30]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-30.liferay.com/job/publish-testray-report/13354/">publish-testray-report#13354</a><br/></div></div>

catarineaguiar commented:
@kyle-miho possible regression:

```
 [exec] 2022-10-25T18:52:42.213-0700 [DEBUG] [org.gradle.launcher.daemon.server.DefaultIncomingConnectionHandler] Starting executing command: Build{id=59dc8341-40c8-4f84-8527-01dac5cc0c99, currentDir=/opt/dev/projects/github/liferay-portal/modules} with connection: socket connection from /127.0.0.1:46071 to /127.0.0.1:37010.
     [exec] 2022-10-25T18:52:42.213-0700 [ERROR] [org.gradle.launcher.daemon.server.DaemonStateCoordinator] Command execution: started DaemonCommandExecution[command = Build{id=59dc8341-40c8-4f84-8527-01dac5cc0c99, currentDir=/opt/dev/projects/github/liferay-portal/modules}, connection = DefaultDaemonConnection: socket connection from /127.0.0.1:46071 to /127.0.0.1:37010] after 0.0 minutes of idle
     [exec] 2022-10-25T18:52:42.213-0700 [INFO] [org.gradle.launcher.daemon.server.DaemonRegistryUpdater] Marking the daemon as busy, address: [dec43585-9494-41d5-acf4-ee7b9056de77 port:46071, addresses:[localhost/127.0.0.1]]
```
```
     [echo] The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test-batch.xml:6145: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test-batch.xml:2376: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test-batch.xml:1392: /opt/dev/projects/github/liferay-portal/build-test-batch.xml:2378: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test-batch.xml:365: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test-batch.xml:2413: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-common.xml:1007: Unable to execute Gradle task: testIntegration.
```
full log [here](https://testray.liferay.com/reports/production/logs/2022-10/test-1-30/test-portal-acceptance-pullrequest(master)/5193/modules-integration-mysql57-jdk8/0/6/jenkins-console.txt.gz?authuser=0)

kyle-miho commented:
Shelved for now, moving to blocked until further notice
